Api/CashModuleApi/CashModuleApi/Controllers/BankAccountController.cs

```diff
@@ -1,43 +1,93 @@
 ﻿using Microsoft.AspNetCore.Mvc;
-
-// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+using CashModuleApi.Domain;
+using CashModuleApi.Services.BankAccounts;
 
 namespace CashModuleApi.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class BankAccountController : ControllerBase
     {
-        // GET: api/<BankAccountController>
+        private readonly IBankAccountService Service;
```

Comment:
private variables should start with _ and small letter(camel case). In this case it should be _bankAccountService